CONTRIBUTING.md

```diff
@@ -4,11 +4,63 @@ Thanks for your interest in helping improve IaSQL! 🎉
 
 If you are looking for IaSQL's documentation, go here instead: https://docs.iasql.com/
 
-This document is for people who want to contribute code to IaSQL. There are also other ways to contribute, such as reporting bugs, creating feature requests, helping other users in our forums, Discord etc.
+This document is for people who want to contribute to IaSQL. There are multiple ways to contribute to the project other than code, such as reporting bugs, creating feature requests, helping other users in online spaces, Discord etc.
+
+## Get Involved
+
+Anyone can become an IaSQL Contributor regardless of skill level, experience, or background. All types of contributions are meaningful. Our membership system was designed to reflect this.
+
+**Anything that supports the IaSQL community is a contribution to the project.** This includes but is not limited to:
+  - Submitting (and Merging) a Pull Request
+  - Filing a Bug Report or Feature Request
+  - Updating Documentation
+  - Answering questions about IaSQL on GitHub or Discord
+  - Answering questions on Discord, Stack Overflow, Twitter, etc.
+  - Blogging, Podcasting, or Livestreaming about IaSQL
+
+## Membership Levels, Roles and Responsibilities
+
+### Contributor L1
+
+Have you done something to contribute to the health, success, or growth of IaSQL? Congratulations, you're officially a contributor!
+
+**Benefits:**
+- Contributor status on the [IaSQL Discord server](https://discord.com/invite/yxNBQugGbH)
+
+**Nomination:**
+- Self nominate by posting the qualifying contribution in Discord (link preferred).
+
+### Contributor L2 (Committer)
+
+**Contributor L2** membership is reserved for users that have shown a commitment to the continued development of the project through ongoing engagement with the community. At this level, contributors are given push access to the project's GitHub repos and must continue to abide by the project's Contribution Guidelines and Code of Conduct.
+
+Anyone who has made several significant (non-trivial) contributions to IaSQL can become a Contributor in recognition of their work. An example of a "significant contribution" might be:
+- ✅ Triaging and supporting non-trivial Discord and GitHub issues
+- ✅ Submitting and reviewing non-trivial PRs
+- ✅ Submitting and reviewing non-trivial documentation edits (multiple sections/pages)
+- ❌ A typo fix, or small documentation edits of only a few sentences
+
+**Responsibilities:**
+- May request write access to relevant IaSQL projects.
```

Comment:
What level of write access are we thinking about here?

Reply:
Oh just being able to merge a PR. I believe in GH `Write` access is needed just to be able to open a PR in the first place by pushing a branch